server/routes/user.tsx

```diff
@@ -6,12 +6,22 @@ import { getUser } from 'server/utils/queryHelpers';
 import { handleErrors } from 'server/utils/errors';
 import { getInitialData } from 'server/utils/initData';
 import { generateMetaComponents, renderToNodeStream } from 'server/utils/ssr';
+import { isUserAffiliatedWithCommunity } from 'server/community/queries';
+// check if member has any pubs or collections for user
```

Comment:
Please remove this comment since it's no longer attached to the function it describes!